frappe/model/db_query.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, doctype, user=None):
 		self.reference_doctype = None
 
 	def execute(self, fields=None, filters=None, or_filters=None,
-		docstatus=None, group_by=None, order_by=None, limit_start=False,
+		docstatus=None, group_by=None, order_by="default_ordering", limit_start=False,
```

Comment:
How about `"DEFAULT_ORDERING"` instead. Feels like a constant rather than an actual value? There's a chance that "default_ordering" could be a fieldname for someone....

I think we could do `"__DEFAULT_ORDERING"` or `"KEEP_DEFAULT_ORDERING"` too.

Reply:
switched to `KEEP_DEFAULT_ORDERING` in #62fc126